docs/developer-docs/frontend/serving-static-assets.mdx

```diff
@@ -0,0 +1,150 @@
+# Serving static assets
+
+## Overview
+
+When a dapp is deployed to ICP, the frontend assets will be displayed via the canister's ID, followed by `.ic0.app`, if the dapp has a frontend configured. For example, the [Motoko Playground](https://m7sm4-2iaaa-aaaab-qabra-cai.ic0.app/) is an application with a frontend user interface that can be used to deploy canisters in a temporary, sandbox environment. This dapp can be accessed via the URL `https://m7sm4-2iaaa-aaaab-qabra-cai.ic0.app/`. 
```

Comment:
Should we make it clear that this is when a dapp is deployed on mainnet?

Reply:
We could also mention that in local deployment the canister is accessible at `http://127.0.0.1:4943/?canisterId=m7sm4-2iaaa-aaaab-qabra-cai`.

Reply:
The `4943` is the default port number and the actual port number may differ if the developer has changed via a dfx flag or config.

Reply:
Added additional info about local deployment versus mainnet deployment URLs.